addons/account/models/account_order_line_mixin.py

```diff
@@ -0,0 +1,111 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, _
+from odoo.tools import format_date
+
+
+class AccountOrderLineMixin(models.AbstractModel):
+    # This class enables down payment support for child records of account.order.mixin that can be down payments
+    _name = 'account.order.line.mixin'
+    _description = 'Account Order Line Mixin'
+
+    is_downpayment = fields.Boolean(string="Is a down payment",
+                                    help="Down payments are made when creating account moves from an order."
+                                         " They are not copied when duplicating an order.",
+                                    )
```

Comment:
```suggestion
    is_downpayment = fields.Boolean(
        string="Is a down payment",
        help="Down payments are made when creating account moves from an order. They are not copied when duplicating an order.",
    )
```

Sale only overrides this field to say "sales order" in the helper ; we don't care, do we ?